python/src/equistore/operations/_dispatch.py

```diff
@@ -65,6 +65,7 @@ def bincount(input, weights=None, minlength=0):
                                         Defaults to None.
         minlength (int, optional): A minimum number of bins for the output array.
                                         Defaults to 0.
+        out_format
```

Comment:
looks like a leftover

Reply:
Could you also add a link to `array_like_data` here? Pointing future devs to this problem =)

Reply:
not sure what you meant but i added a comment

Reply:
👍